website/addons/s3/tests/factories.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+"""Factory boy factories for the Box addon."""
+import mock
+from datetime import datetime
+from dateutil.relativedelta import relativedelta
```

Comment:
It looks like `mock`, `datetime` and `relativedelata` are not used and can be removed